docusign/1.0/modules/ROOT/pages/docusign-connector-examples.adoc

```diff
@@ -0,0 +1,216 @@
+account path on `localhost`, port `8081`.
+
+This example uses variables for some field values. You can either:
+
+* Replace the variables with their values in the code.
+* Provide the values for each variable in a properties file and then refer to
+that file from the connector configuration.
+
+If you don't know how to use a properties file,
+see xref:mule-runtime::mule-app-properties-to-configure.adoc[Configuring Property Placeholders].
+
+To configure HTTP *Listener*:
+
+. Create a new Mule project in Studio.
+. In *Mule Palette*, search for `http` and select the *Listener* operation:
+. Drag the *Listener* operation onto the canvas.
+. In the *Listener* configuration, click the *Add* icon next to the *Connector configuration*
+field to add a global element.
+. Click *OK* to accept the defaults.
+. Set the *Path* field to `/`:
+
+=== Add the Parse Template Component
+
+The *Parse Template* component creates a template for an HTML form (`form.html`),
+which is located in the main resources folder. The example uses this form to
+convey information for creating an envelope.
+
+. From the Mule Palette view, search for `parse` and drag *Core> Parse Template* onto the canvas, next to
+HTTP *Listener*.
+. Configure the following fields in the *Publish* properties window:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Display Name |Name for the component, such as `Parse Template`.
+|Location |Location of the file, such as `form.html`.
+|===
+
+== Create the Second Flow
+
+The second flow creates the actual DocuSign envelope. The `status`, `emailSubject`,
+`documents`, and `recipients` (combined URI and recipient URI) parameters must be provided.
+
+image::docusign-second-flow.png[DocuSign Connector Second Flow in Studio]
+
+In this flow, you configure:
+
+* An HTTP *Listener* component
+* *Logger* component
+* *Envelopes Post Envelopes* operation
+* *Transform Message* component
+* A second *Logger* component
+
+=== Configure HTTP Listener
+
+Configure HTTP *Listener* to initiate a Mule flow when a call is made to the `/`
+account path on `localhost`, port `8081`.
+
+This example uses variables for some field values. You can either:
+
+* Replace the variables with their values in the code.
+* Provide the values for each variable in a properties file and then refer to
+that file from the connector configuration.
+
+If you don't know how to use a properties file,
+see xref:mule-runtime::mule-app-properties-to-configure.adoc[Configuring Property Placeholders].
+
+To configure HTTP *Listener*:
+
+. Create a new Mule project in Studio.
+. In *Mule Palette*, search for `http` and select the *Listener* operation:
+. Drag the *Listener* operation onto the canvas.
+. In the *Listener* configuration, click the *Add* icon next to the *Connector configuration*
+field to add a global element.
+. Click *OK* to accept the defaults.
+. Set the *Path* field to `/create-envelope`:
+
+=== Add a Logger Component
```

Comment:
If possible, add short description of what the component does, like you have for the other components.